src/components/deck/column-common.module.scss

```
@@ -224,6 +224,19 @@
   cursor: pointer;
 }
 
+/* クエリが暴走で打ち切られ保留中。押すと再開する (#783 V15) */
+.querySuspendedBanner {
+  composes: statusBanner;
```

Stylelint (17.14.1)

[error] Expected "statusBanner" to be "statusbanner" (value-keyword-case)

(value-keyword-case)